### PR TITLE
Update Subgraph for Studio Compatibility

### PR DESCRIPTION
**Pull request by nlordell**
This PR updates the Subgraph to the latest `graph` CLI for Subgraph Studio compatibility. There were some minor code changes needed to make the tests pass and the Subgraph to index.

### Test Plan

I was also able to deploy the Subgraph to my personal Subgraph Studio account and use the API to make requests to it.

```
$ curl -s localhost:8080/supply | jq
{
  "total": "1000000000000000000000000000",
  "circulating": "82734171815734095599344818"
}
```

### CODE REVIEW

File: src/mapping.ts [hunk 1]
```diff
@@ -16,6 +16,7 @@ export function loadOrCreateHolder(address: Address): Holder {
   let holder = Holder.load(address.toHex());
   if (!holder) {
     holder = new Holder(address.toHex());
+    holder.balance = BigInt.zero();
```
Comment (bh2smith) [inline src/mapping.ts:19]: So we have to initialize zeroes now?

**Review by bh2smith — APPROVED** [review]
Looks good! Does this mean I should redeploy when merged?

File: src/mapping.ts (hunk 1 above)
Comment (nlordell) [reply src/mapping.ts:19]: Yes, otherwise the indexing errors out. Seems to be a breaking Graph change.

**Comment by nlordell** [comment]
> Does this mean I should redeploy when merged?

No, we want to deploy in Subgraph Studio. I already have a deployment under my account. Let see if we use the API seldomly enough so that we don't need to pay GRT tokens.

**Pull request merged**